src/api/common/schemas/natural.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "natural",
+  "description": "A non-negative integer",
+  "type": "integer",
+  "minimum": 0
```

Comment:
First ledger sequence is 0 according to the wiki https://wiki.ripple.com/Genesis_ledger